### PR TITLE
[Sprint 4] Feature book detail ui #2

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/137007
- Feature book detail ui #2 
- Effect : None

### CODE REVIEW

File: app/src/main/java/com/framgia/fbook/screen/bookdetail/ItemUserClickListener.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.framgia.fbook.screen.bookdetail
+
+import com.framgia.fbook.data.model.User
+
+/**
+ * Created by framgia on 14/09/2017.
+ */
+interface ItemUserClickListener {
+  fun onItemUserClick(user: User)
+}
```
Comment (nguyenminhduong) [inline app/src/main/java/com/framgia/fbook/screen/bookdetail/ItemUserClickListener.kt]: missing end of line

File: app/src/main/java/com/framgia/fbook/screen/bookdetail/ItemUserViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Created by framgia on 14/09/2017.
+ */
+class ItemUserViewModel(val mUser: User,
+    private val mItemUserClickListener: ItemUserClickListener?) : BaseObservable() {
+  fun onItemUserClick(view: View) {
+    mItemUserClickListener?.onItemUserClick(mUser)
+  }
+}
```
Comment (nguyenminhduong) [inline app/src/main/java/com/framgia/fbook/screen/bookdetail/ItemUserViewModel.kt]: missing end of line

File: app/src/main/res/drawable/background_hexagon.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="@dimen/dp_40"
+    android:viewportHeight="15.993"
+    android:viewportWidth="15.993"
+    android:width="@dimen/dp_40">
+
+    <path
+        android:fillColor="@color/colorPrimary"
+        android:pathData="M 7.99 0 L 0.016 6.039 L 4.01 15.993 L 11.963 15.993 L 15.976 6.039 Z"/>
+</vector>
```
Comment (nguyenminhduong) [inline app/src/main/res/drawable/background_hexagon.xml]: missing end of line

File: app/src/main/res/layout/item_list_user.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dp_5"
+            android:gravity="center"
+            android:textSize="@dimen/sp_15"
+            />
+    </LinearLayout>
+</layout>
```
Comment (nguyenminhduong) [inline app/src/main/res/layout/item_list_user.xml]: missing end of line

File: app/src/main/res/layout/activity_bookdetail.xml [hunk 5]
```diff
@@ -96,158 +98,398 @@
...
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:layout_marginTop="@dimen/dp_5"
+                                android:rating="@{viewModel.mBook.avgStar}"
+                                app:layout_constraintLeft_toLeftOf="parent"
+                                app:layout_constraintTop_toBottomOf="@id/text_title_count_view"
+                                style="@style/Base.Widget.AppCompat.RatingBar.Indicator"
+                                />
+                        </android.support.constraint.ConstraintLayout>
+                        <LinearLayout
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_bookdetail.xml]: WARNING: [DisableBaselineAlignment] Set `android:baselineAligned="false"` on this element for better performance
 When a LinearLayout is used to distribute the space proportionally between nested layouts, the baseline alignment property should be turned off to make the layout computation faster.

File: app/src/main/res/layout/activity_bookdetail.xml [hunk 6]
```diff
@@ -96,158 +98,398 @@
...
+                        android:layout_marginRight="@dimen/dp_20"
+                        android:layout_marginStart="@dimen/dp_20"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:text="@string/activities"
+                        android:textAllCaps="true"
+                        android:textColor="@color/color_dark_gray"
+                        android:textSize="@dimen/sp_17"
+                        android:textStyle="bold"
+                        />
+                    <LinearLayout
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_bookdetail.xml]: WARNING: [DisableBaselineAlignment] Set `android:baselineAligned="false"` on this element for better performance
 When a LinearLayout is used to distribute the space proportionally between nested layouts, the baseline alignment property should be turned off to make the layout computation faster.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -33,6 +40,13 @@
     <string name="import_data_from_google_book">Import data from google books</string>
     <string name="can_find_data">Can\'t find data</string>
     <string name="you_must_be_login_into_perform_this_function">You must be login in to perform this function. Please login !</string>
+    <string name="want_to_read">want to read</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: Pls uppercase first character of values

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -33,6 +40,13 @@
...
+    <string name="author_two_dots">Author:</string>
+    <string name="add_owner_this_book">add onwer this book</string>
+    <string name="remove_owner_this_book">remove onwer this book</string>
+    <string name="cancel_waiting_this_book">cancel waiting this book</string>
+    <string name="count_view">Count view:</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: -> View counts

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -33,6 +40,13 @@
     <string name="import_data_from_google_book">Import data from google books</string>
     <string name="can_find_data">Can\'t find data</string>
     <string name="you_must_be_login_into_perform_this_function">You must be login in to perform this function. Please login !</string>
+    <string name="want_to_read">want to read</string>
+    <string name="author_two_dots">Author:</string>
+    <string name="add_owner_this_book">add onwer this book</string>
+    <string name="remove_owner_this_book">remove onwer this book</string>
+    <string name="cancel_waiting_this_book">cancel waiting this book</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: llike above

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -33,6 +40,13 @@
     <string name="import_data_from_google_book">Import data from google books</string>
     <string name="can_find_data">Can\'t find data</string>
     <string name="you_must_be_login_into_perform_this_function">You must be login in to perform this function. Please login !</string>
+    <string name="want_to_read">want to read</string>
+    <string name="author_two_dots">Author:</string>
+    <string name="add_owner_this_book">add onwer this book</string>
+    <string name="remove_owner_this_book">remove onwer this book</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: llike above

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -33,6 +40,13 @@
     <string name="import_data_from_google_book">Import data from google books</string>
     <string name="can_find_data">Can\'t find data</string>
     <string name="you_must_be_login_into_perform_this_function">You must be login in to perform this function. Please login !</string>
+    <string name="want_to_read">want to read</string>
+    <string name="author_two_dots">Author:</string>
+    <string name="add_owner_this_book">add onwer this book</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: llike above

File: app/src/main/res/values-vi/strings.xml [hunk 12]
```diff
@@ -29,6 +29,12 @@
     <string name="following">Đang theo dõi</string>
     <string name="add_your_book">Thêm sách của bạn</string>
     <string name="can_find_data">Không tìm thấy dữ liệu</string>
+    <string name="want_to_read">Muốn đọc</string>
+    <string name="author_two_dots">Tác giả:</string>
+    <string name="count_view">Số lượt xem:</string>
```
Comment (daolq3012) [inline app/src/main/res/values-vi/strings.xml]: -> Lượt xem:

**Pull request merged**